FrostySdk/Managers/AssetManager.cs

```diff
@@ -881,7 +881,7 @@ public void Reset()
                     RevertAsset(entry, suppressOnModify: false);
             }
 
-            foreach(BundleEntry bundle in bundles.Where(b => b.Added))
+            foreach(BundleEntry bundle in bundles.Where(b => b.Added).ToList())
```

Comment:
i think this would probably be cleaner
```suggestion
            bundles.RemoveAll(b => b.Added);
```